app/models/concerns/exportable.rb

```diff
@@ -48,7 +48,10 @@ module Exportable
     "Gestation at procedure in weeks" => :gestation_at_procedure,
     "Procedure cost" => :procedure_cost_amount,
     "Check number" => :check_number,
-    "Date of Check" => :date_of_check
+    "Date of Check" => :date_of_check,
+
+    # Notes
+    "Notes Count" => :notes_count,
```

Comment:
I have a left field question -- after taking the export for a test drive, I found myself sort of wishing for an indicator of whether a patient is archived or not. If you don't think it presents a PII or other problem, what are your thoughts about adding it?